dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,331 @@
+                   + "                                               WHEN 10 THEN 'SECOND'"
+                   + "                                               WHEN 11 THEN 'FRACTION(1)'"
+                   + "                                               WHEN 12 THEN 'FRACTION(2)'"
+                   + "                                               WHEN 13 THEN 'FRACTION(3)'"
+                   + "                                               WHEN 14 THEN 'FRACTION(4)'"
+                   + "                                               WHEN 15 THEN 'FRACTION(5)'"
+                   + "                                           END"
+                   + "            WHEN   11 THEN 'BYTE'"
+                   + "            WHEN   12 THEN 'TEXT'"
+                   + "            WHEN   13 THEN 'VARCHAR'"
+                   + "            WHEN   14 THEN 'INTERVAL'"
+                   + "            WHEN   15 THEN 'NCHAR'"
+                   + "            WHEN   16 THEN 'NVARCHAR'"
+                   + "            WHEN   17 THEN 'INT8'"
+                   + "            WHEN   18 THEN 'SERIAL8'"
+                   + "            WHEN   19 THEN 'SET'"
+                   + "            WHEN   20 THEN 'MULTISET'"
+                   + "            WHEN   21 THEN 'LIST'"
+                   + "            WHEN   22 THEN 'ROW'"
+                   + "            WHEN   23 THEN 'COLLECTION'"
+                   + "            WHEN   24 THEN 'ROWDEF'"
+                   + "            WHEN   40 THEN 'LVARCHAR'"
+                   + "            WHEN   43 THEN 'LVARCHAR'"
+                   + "            WHEN   45 THEN 'BOOLEAN'"
+                   + "            WHEN   52 THEN 'BIGINT'"
+                   + "            WHEN   53 THEN 'BIGSERIAL'"
+                   + "            WHEN 2061 THEN 'IDSSECURITYLABEL'"
+                   + "            WHEN 4118 THEN 'ROW'"
+                   + "        END"
+                   + "           AS data_type"
+                   + "     , " + (forProcedureParameters ? "c.paramattr" : "1") // Dummy value for table/view columns.
+                   + "           AS direction"
+                   + "     , c." + (forProcedureParameters ? "paramid" : "colno")
+                   + "           AS position"
+                   + "  FROM informix." + (forProcedureParameters ? "sysprocedures p" : "systables p")
+                   + "     , informix." + (forProcedureParameters ? "sysproccolumns c" : "syscolumns c")
+                   + " WHERE p." + (forProcedureParameters ? "procid" : "tabid") + " = c." + (forProcedureParameters ? "procid" : "tabid")
+                   + "   AND ";
+
+        if (qualifiers.length == 2) {
+            qry += "LOWER(p.owner) = ? AND LOWER(p." + (forProcedureParameters ? "procname" : "tabname") + ") = ?";
+        } else {
+            qry += "LOWER(p." + (forProcedureParameters ? "procname" : "tabname") + ") = ?";
```

Comment:
In both cases of this else statements a large fraction of code seems the same. I wonder if we can make it look a bit simpler.